cheroot/ssl/builtin.py

```diff
@@ -43,20 +43,25 @@ class BuiltinSSLAdapter(Adapter):
     """The ssl.SSLContext that will be used to wrap sockets where available
     (on Python > 2.7.9 / 3.3)
     """
+    ciphers = None
+    """The ciphers list of SSL."""
```

Comment:
Yes, attribute docstrings exist. I endorse them despite the modest adoption in the wild.